picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -664,13 +668,24 @@ void shutdown() {
     private ExecutorService service;
     private Cache cache;
     private Listener listener;
-    private RequestTransformer transformer;
-    private List<RequestHandler> requestHandlers;
+    private RequestTransformer requestTransformer;
+    private List<RequestHandler> requestHandlers = new ArrayList<>();
```

Comment:
@JakeWharton should I expose `requestHandlers` via a getter so verify it's contents in the `newBuilder` test?